docs/kratos/social-signin/83_linkedin.mdx

```diff
@@ -0,0 +1,145 @@
+9. Copy the **Client ID** from the registered application in LinkedIn to the **Client ID** field in the form in the Ory Console.
+10. In the **Scopes** field of the form in the Ory Console, add the following scopes:
+    - `r_emailaddress`
+    - `r_liteprofile`
+11. In the **Data Mapping** field of the form in the Ory Console, add the following Jsonnet code snippet,
+    which maps the desired claims to the Ory Identity schema:
+
+   ```jsonnet
+   local claims = {
+     email_verified: false,
+   } + std.extVar('claims');
+   {
+     identity: {
+       traits: {
+         // Allowing unverified email addresses enables account
+         // enumeration attacks,  if the value is used for
+         // verification or as a password login identifier.
+         //
+         // Therefore we only return the email if it (a) exists and (b) is marked verified
+         // by LinkedIn.
+         [if 'email' in claims && claims.email_verified then 'email' else null]: claims.email,
+       },
+     },
+   }
+   ```
+
+   ```mdx-code-block
+   <JsonnetWarning format="Jsonnet code snippets" use="data mapping" />
+   ```
+
+12. Click **Save Configuration**.
+
+</TabItem>
+<TabItem value="cli" label="Ory CLI">
+
+Follow these steps to add LinkedIn as a social sign-in provider to your project using the Ory CLI:
+
+1. [Create a LinkedIn application](https://developer.linkedin.com/), verify it and [activate **Sign In with LinkedInV2**](https://www.linkedin.com/pulse/how-get-signin-linkedin-work-taric-andrade/).
+2. In the created app, add this Redirect URI to the **Authorized redirect URLs** in the LinkedIn Auth configuration:
+
+   ```shell
+   https://{project.slug}.projects.oryapis.com/self-service/methods/oidc/callback/linkedin
+   ```
+
+3. Create a Jsonnet code snippet to map the desired claims to the Ory Identity schema.
+
+   ```jsonnet
+   local claims = {
+     email_verified: false,
+   } + std.extVar('claims');
+   {
+     identity: {
+       traits: {
+         // Allowing unverified email addresses enables account
+         // enumeration attacks,  if the value is used for
+         // verification or as a password login identifier.
+         //
+         // Therefore we only return the email if it (a) exists and (b) is marked verified
+         // by LinkedIn.
+         [if 'email' in claims && claims.email_verified then 'email' else null]: claims.email,
+       },
+     },
+   }
+   ```
+
+   ```mdx-code-block
+   <JsonnetWarning format="Jsonnet code snippets" use="data mapping" />
+   ```
+
+4. Encode the Jsonnet snippet with [Base64](https://www.base64encode.org/) or host it under an URL accessible to Ory Network.
+
+   ```shell
+   cat your-data-mapping.jsonnet | base64
+   ```
+
+5. Download the Ory Identities config from your project and save it to a file:
+
+   ```shell
+   ## List all available projects
+   ory list projects
+   ## Export the project id
+   export project_id={project-id}
```

Comment:
Unnecessary step, please remove it.

Reply:
Tbh if you include this you can just copy & paste the code example, if you just have 
`ory get identity-config {project-id} --format yaml > identity-config.yaml`
you need to change the code and you cant just copy&paste it

I change it so it is the same as with the other tutorials, but if you export the project ID its much more practical for everyday use IMO.